code/game/objects/items/violin.dm

```diff
@@ -1,393 +1,7 @@
-//copy pasta of the space piano, don't hurt me -Pete
-
 /obj/item/violin
 	name = "space violin"
 	desc = "A wooden musical instrument with four strings and a bow. \"The devil went down to space, he was looking for an assistant to grief.\""
 	icon = 'icons/obj/musician.dmi'
 	icon_state = "violin"
 	item_state = "violin"
-	force = 10
-	var/datum/song/song
-	var/playing = 0
-	var/help = 0
-	var/edit = 1
-	var/repeat = 0
-	var/max_lines = 50
-
-/obj/item/violin/proc/playnote(var/note as text)
-	//to_chat(world, "Note: [note]")
-	var/soundfile
-	/*BYOND loads resource files at compile time if they are ''. This means you can't really manipulate them dynamically.
-	Tried doing it dynamically at first but its more trouble than its worth. Would have saved many lines tho.*/
-	switch(note)
-		if("Cn1")	soundfile = 'sound/violin/Cn1.mid'
-		if("C#1")	soundfile = 'sound/violin/C#1.mid'
```

Comment:
There's a ton of sounds here that could be removed from the code if we are removing the violin's functionality.

Reply:
you cant cut back on violins you'll regret this

Reply:
The sounds already don't exist

Reply:
You've made an enemy for life.